### ISSUE TITLE
Activate extra destination fields

### ISSUE THREAD

**Issue by AntMadeira** [openstreetmap/id-tagging-schema#1065]
### OSM Tag(s)

destination:colour=* + destination:symbol=*

### How would you like this tag to see supported?

As a Field

### Label

Destination colour + destination symbol

### Link to OSM Wiki page

https://wiki.openstreetmap.org/wiki/Key:destination:symbol

### Status of the Tag

In Use

### Usage of the tag

55,703

### Further Information

Hi there.
As stated [here](https://github.com/openstreetmap/iD/issues/10006) first, many iD mappers don't know that destination:symbol and destination:colour exists, so why doesn't iD help with that?
My suggestion is to activate those fields whenever the destination field is filled with one or more values.
I think this would be a great addition to add more useful info to highway links, and also would help mappers to investigate further other more complex info, like lanes, :forward and :backward keys.

Regards.

**Comment by 1ec5** [comment]
`destination:symbol` is already supported on `highway=*_link` and `type=destination_sign` presets, but it’s an optional field rather than one that appears by default:

https://github.com/openstreetmap/id-tagging-schema/blob/ef73d649b6bdaad0a8bccd6ab25ed341c5d986e1/data/fields/destination/symbol.json#L4

**Comment by AntMadeira** [comment]
But the field only appears if you enter the tag manually. What I suggest is to make it appear as soon as you enter a destination, which is very different and more helpful.

BTW, there's also a bug when entering multiple `none` values to the `destination:symbol` key. If we add two of more `none` values separated with `;` iD only considers one. And if you grab and move one of those `none` values, iD eliminates all of them and leaves only one in the value part of the tag.

**Comment by matkoniecz** [comment]
looks like it should be unblocked? After https://github.com/openstreetmap/schema-builder/pull/87

**Comment by matkoniecz** [comment]
Nope, https://github.com/openstreetmap/schema-builder/pull/87 allow multiple values - not multiple tags
